tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/IProcessConfigurationService.cs

```diff
@@ -0,0 +1,45 @@
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Models.Responses.Package;
+using Azure.Sdk.Tools.Cli.Helpers;
+
+namespace Azure.Sdk.Tools.Cli.Services;
+
+/// <summary>
+/// Service for creating process options based on configuration and executing processes with standardized response handling.
+/// </summary>
+public interface IProcessConfigurationService
```

Comment:
This doesn't seem like on a configuration it also includes execution. 

Also, these seem like they might be more interesting on the SpecGenSdkConfigHelpers classes instead.

Reply:
Combined this class to the `SpecGenSdkConfigHelpers` class. It's a bit heavier to contain process execution method, but I will leave it for now since this will be replaced by the new solution.